reflex/.templates/web/components/reflex/radix_themes_color_mode_provider.js

```diff
@@ -5,7 +5,7 @@ import { ColorModeContext, defaultColorMode } from "/utils/context.js";
 export default function RadixThemesColorModeProvider({ children }) {
   const { theme, resolvedTheme, setTheme } = useTheme();
   const [rawColorMode, setRawColorMode] = useState(defaultColorMode);
-  const [resolvedColorMode, setResolvedColorMode] = useState(theme);
+  const [resolvedColorMode, setResolvedColorMode] = useState("dark");
```

Comment:
Is this right we want to default to dark?

Reply:
yes it's for the pre-rendering to be consistant and avoid hydration error.
It will not impact the actual render of the page in the browser.